apiutils/task.go

```diff
@@ -0,0 +1,21 @@
+package apiutils
```

Comment:
> > +package apiutils
> (it's literally 4 lines and it's best practice to just copy around small functions)
> 

Hmm, I disagree. This question keeps coming up because we don't have a place to put code that's common to the agent and a manager. Let's start the package instead of copying code every time.